pkg/apis/build/v1alpha1/build_types.go

```diff
@@ -54,11 +54,19 @@ type BuildSpec struct {
 	CacheName      string                      `json:"cacheName"`
 	Env            []corev1.EnvVar             `json:"env"`
 	Resources      corev1.ResourceRequirements `json:"resources"`
+	Rebase         *RebaseConfig               `json:"rebase,omitempty"`
+}
+
+type RebaseConfig struct {
+	ImageRef         string `json:"imageRef"`
```

Comment:
This `ImageRef` name fells hard to understand what it is about, maybe `ImageToRebaseRef`?

Reply:
Makes sense

Reply:
turns out we dont need it so :)